examples/tv-casting-app/android/App/app/src/main/jni/com/chip/casting/TvCastingApp.java

```diff
@@ -29,6 +29,46 @@ public native boolean openBasicCommissioningWindow(
 
   public native boolean sendUserDirectedCommissioningRequest(String address, int port);
 
+  /**
+   * Send a request to Discover commissioners using DNS-SD
+   *
+   * @param discoverySuccessCallback
+   * @param discoveryFailureCallback
+   * @return true, if the discovery request was sent successfully, false otherwise
+   */
+  public boolean discoverCommissioners(
+      SuccessCallback<DiscoveredNodeData> discoverySuccessCallback,
+      FailureCallback discoveryFailureCallback);
+
+  /**
+   * Send a User Directed Commissioning request to a commissioner
+   *
+   * @param commissioner
+   * @return
+   */
+  public boolean sendCommissioningRequest(DiscoveredNodeData commissioner);
+
+  /**
+   * @return List of VideoPlayers actively connected and available for targeting. If no such Video
+   *     Players are found, this returns an empty list
+   */
+  public native List<VideoPlayer> getActiveTargetVideoPlayers();
+
+  /**
+   * @return Read all VideoPlayers the TvCastingApp knows about, from previous connections (may not
+   *     be actively connected to some/any of them)
+   */
+  public native List<VideoPlayer> readAllVideoPlayers();
```

Comment:
could we follow same verb for the method above and this one. So either get for both or read for both?

Reply:
I used the verb "read" here deliberately to tell the client that we are reading from the app's cache in this API, rather than "getting" the list of video players over the network.